picard/ui/options/maintenance.py

```diff
@@ -0,0 +1,178 @@
+        self.ui.tableWidget.setRowCount(len(orphan_options))
+        for row, option_name in enumerate(sorted(orphan_options)):
+            tableitem = QtWidgets.QTableWidgetItem(option_name)
+            tableitem.setData(QtCore.Qt.UserRole, option_name)
+            tableitem.setFlags(tableitem.flags() | QtCore.Qt.ItemIsUserCheckable)
+            tableitem.setCheckState(False)
+            self.ui.tableWidget.setItem(row, 0, tableitem)
+            tableitem = QtWidgets.QTextEdit()
+            text = self.make_setting_value_text(option_name)
+            tableitem.setPlainText(text)
+            tableitem.setReadOnly(True)
+            # Adjust row height to reasonably accommodate values with more than one line, with a minimum
+            # height of 25 pixels.  Long line or multi-line values will be expanded to display up to 5
+            # lines, assuming a standard line height of 18 pixels.  Long lines are defined as having over
+            # 50 characters.
+            text_rows = max(text.count("\n") + 1, int(len(text) / 50))
+            row_height = max(25, 18 * min(5, text_rows))
+            self.ui.tableWidget.setRowHeight(row, row_height)
+            self.ui.tableWidget.setCellWidget(row, 1, tableitem)
+        self.ui.tableWidget.resizeColumnsToContents()
+        self.ui.select_all.setCheckState(False)
+        self.enable_cleanup_changed()
+
+    def column_items(self, column):
+        for idx in range(self.ui.tableWidget.rowCount()):
+            yield self.ui.tableWidget.item(idx, column)
+
+    def selected_options(self):
+        for item in self.column_items(0):
+            if item.checkState() == QtCore.Qt.Checked:
+                yield item.data(QtCore.Qt.UserRole)
+
+    def select_all_changed(self):
+        state = self.ui.select_all.checkState()
+        for item in self.column_items(0):
+            item.setCheckState(state)
+
+    def save(self):
+        if not self.ui.enable_cleanup.checkState() == QtCore.Qt.Checked:
+            return
+        to_remove = set(self.selected_options())
+        if to_remove and QtWidgets.QMessageBox.question(
+            self,
+            _('Confirm Remove'),
+            _("Are you sure you want to remove the selected option settings?"),
+        ) == QtWidgets.QMessageBox.Yes:
+            config = get_config()
+            for item in to_remove:
+                Option.add_if_missing('setting', item, None)
+                log.warning("Removing option setting '%s' from the INI file.", item)
+                config.setting.remove(item)
+
+    def make_setting_value_text(self, key):
+        config = get_config()
+        value = config.setting.raw_value(key)
+        return repr(value)
+
+    # def select_all_changed(self):
+    #     state = self.ui.select_all.checkState()
+    #     for idx in range(self.ui.tableWidget.rowCount()):
+    #         item = self.ui.tableWidget.item(idx, 0)
+    #         item.setCheckState(state)
```

Comment:
@rdswift you can remove those lines

Reply:
Crap!  I thought I had removed the old method, but I guess I had just commented it out.  Sorry about that.